docker/README.md

```diff
@@ -0,0 +1,38 @@
+# Cygnus Containerization
+
+The process of building this application and packing it into a container could be a little long in time terms. Addressing this issue, the process is separated in two steps: Building the jar and packing the cygnus container.
+
+## Build the jar
+
+Run this command from the root of this repository
+
+	docker-compose -f ./docker/0.compose.jar-compiler.yml -p cygnus run --rm compiler
```

Comment:
Please, indent this line by adding 4 white spaces. This will do Markdown prints this line in "terminal"-style. Do the same with other commands appearing in the document.